django_36/purchase/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+from users.models import Users
+from book.models import Books
+
+
+class Purchases(models.Model):
+    user = models.ForeignKey(Users, on_delete=models.CASCADE)
+    book = models.ForeignKey(Books, on_delete=models.CASCADE, default='')
```

Comment:
nit: I think empty value should be null, so it should be like black=True, null=True